.travis.yml

```diff
@@ -11,6 +15,14 @@ branches:
 
 install:
   - npm ci
+  - git clone https://github.com/DefinitelyTyped/DefinitelyTyped ../DefinitelyTyped
+
+script:
+  - npm tes
+  - npm run build
+  - npm run clean
+  - npm run parse
```

Comment:
This relies on a cache to avoid querying npm for everything. It is NOT the most reliable cache, and I don't understand travis' caching strategy. You should test to see whether it's running `npm info` on every `@types` package for every CI run. If it's *not*, then you need to make sure that it's correctly running `npm info` on packages that were recently updated in DT.